### PR TITLE
Add speech v1 configs

### PR DESCRIPTION
**Pull request by shinfan**

### CODE REVIEW

File: google/cloud/speech/v1/cloud_speech_gapic.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+interfaces:
+- name: google.cloud.speech.v1.Speech
+  collections: []
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
```
Comment (geigerj) [inline google/cloud/speech/v1/cloud_speech_gapic.yaml]: `UNAVAILABLE`?

File: google/cloud/speech/v1/cloud_speech_gapic.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.speech.spi.v1
+  python:
+    package_name: google.cloud.speech.v1
```
Comment (geigerj) [inline google/cloud/speech/v1/cloud_speech_gapic.yaml]: `google.cloud.gapic.speech.v1`

File: google/cloud/speech/v1/cloud_speech_gapic.yaml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 20000
```
Comment (geigerj) [inline google/cloud/speech/v1/cloud_speech_gapic.yaml]: I think @lukesneeringer determined that this should be `190000` based on https://github.com/googleapis/googleapis/pull/297

File: google/cloud/speech/cloud_speech.yaml [hunk 4]
```diff
@@ -12,6 +12,8 @@ documentation:
 
 apis:
 - name: google.cloud.speech.v1beta1.Speech
+- name: google.cloud.speech.v1.Speech
+- name: google.longrunning.Operations
```
Comment (pongad) [inline google/cloud/speech/cloud_speech.yaml]: The `v1beta1.Speech` and `Operations` should be removed.

Comment (shinfan) [inline google/cloud/speech/cloud_speech.yaml]: Done

File: google/cloud/speech/v1/cloud_speech_gapic.yaml (hunk 3 above)
Comment (shinfan) [inline google/cloud/speech/v1/cloud_speech_gapic.yaml]: Done

File: google/cloud/speech/v1/cloud_speech_gapic.yaml (hunk 1 above)
Comment (shinfan) [inline google/cloud/speech/v1/cloud_speech_gapic.yaml]: Done

File: google/cloud/speech/v1/cloud_speech_gapic.yaml (hunk 2 above)
Comment (shinfan) [inline google/cloud/speech/v1/cloud_speech_gapic.yaml]: Done

**Comment by shinfan** [comment]
@pongad @geigerj  PTAL

File: gapic/api/artman_speech_v1.yaml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+common:
```
Comment (geigerj) [inline gapic/api/artman_speech_v1.yaml]: Since there's no longer a service config for `v1beta1`, can we go ahead and also remove the artman/GAPIC configs?

**Comment by garrettjonesgoogle** [comment]
I believe that the `v1beta1` configuration is still needed until all generated clients based on it are deleted. Otherwise, we won't be able to do full cross-API regeneration; the `v1beta` client would not get updated.

**Comment by geigerj** [comment]
In that case, let's preserve a copy of the `v1beta1` service config.

**Comment by lukesneeringer** [comment]
We should keep it until everyone has upgraded, then kill it.

File: gapic/api/artman_speech_v1.yaml (hunk 5 above)
Comment (shinfan) [inline gapic/api/artman_speech_v1.yaml]: Obsolete.

**Comment by shinfan** [comment]
I have made two copies of service yaml for each version. PTAL

**Pull request merged**